packages/vue/src/components/molecules/SfCard/SfCard.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="sf-card">
+    <!-- @slot Custom content that will replace default image-->
+    <slot name="image" v-bind="image">
```

Comment:
I don't think we need to provide slot for image here, since the card itself should render image. If there is no image pass, then sfimage shouldn't be rendered.